tools/parse_soundfont.py

```diff
@@ -1096,8 +1101,11 @@ def process_oot_instruments(self):
                     
     def process_percussion_set(self):
         for drum in self.processed_percussions:
-            start = drum.lowrange - 9
-            end = drum.maxrange - 9
+            # Is there a specific reason to subtract X from the range?
+            # The tuning calculation accepts MIDI note values by default, using "Zelda" note values
+            # or any other note value does nothing but add onto the calculation needlessly...
+            start = drum.lowrange #-9
+            end = drum.maxrange #-9
             index = start
```

Comment:
if you want to remove the " - 21" statements above index should be start - 21 here then